app/code/Magento/Customer/Ui/Component/Listing/Columns.php

```diff
@@ -129,8 +144,11 @@ public function addColumn(array $attributeData, $columnName)
     }
 
     /**
+     * Update Column
+     *
      * @param array $attributeData
-     * @param string $newAttributeCode
+     * @param $newAttributeCode
```

Comment:
Please revert this change as it removes declared param type

Reply:
Resolved